lib/mj-page.js

```diff
@@ -275,7 +275,6 @@ function ConfigureMathJax() {
                 {prev.parentNode.removeChild(prev)}
               //
               //  Add the span, and a div if in display mode,
-              //  then set the role and mark it as being processed
```

Comment:
You took a bit too much.  The "and mark it as being processed" should have been kept.  I suppose it is not critical, but then the comma at the end of the precious line should be removed.